homeassistant/helpers/template.py

```diff
@@ -916,6 +916,30 @@ def device_entities(hass: HomeAssistant, _device_id: str) -> Iterable[str]:
     return [entry.entity_id for entry in entries]
 
 
+def integration_entities(
+    hass: HomeAssistant, entry_name: str, platform: str | None = None
```

Comment:
I'm not to keep on the platform filter.

While it might be helpful, none of the existing methods provide that and I'm not sure if we should add it to all others as well.

Reply:
I was about to PR the platform filter for the other helpers too 😉

Continue or remove? Users can also use expand helper for this but this syntax is cleaner and more efficient

Reply:
I would say remove.

Reply:
done. Can be PR'ed later for all helpers like this if there's interest